ichnaea/log.py

```diff
@@ -25,29 +26,43 @@ def configure_logging():
     logging_level = settings("logging_level")
 
     if local_dev_env:
-        handlers = ["console"]
+        handlers = ["dev"]
+        # Prepare structlog logs for local dev ProcessorFormatter
+        structlog_fmt_prep = structlog.stdlib.ProcessorFormatter.wrap_for_formatter
     else:
         handlers = ["mozlog"]
-
+        # Prepare structlog logs for JsonLogFormatter
+        structlog_fmt_prep = structlog.stdlib.render_to_log_kwargs
```

Comment:
A different final processor is used in non-dev deployments. ``render_to_log_kwargs`` moves the structured data to the `extra` dictionary, so that stdlib `logging` formatters, like `dockerflow.logging.JsonLogFormatter`, can do the Right Thing with logs generated by `structlog`. Logs generated with the stdlib `logging` continue to work as they did before.